groups/instance/data.tf

```diff
@@ -17,6 +17,10 @@ data "vault_generic_secret" "security_s3_buckets" {
   path = "aws-accounts/security/s3"
 }
 
+data "vault_generic_secret" "security_efs_kms_keys_data" {
+  path = "aws-accounts/shared-services/kms"
+}
```

Comment:
We should be using a variable in the vault `path` so it's more dynamic. We can use `account_name` for this.

E.g.:
```
data "vault_generic_secret" "security_efs_kms_keys_data" {
  path = "aws-accounts/${var.account_name}/kms"
}